### PR TITLE
Static header and footer using Bootstrap 3 and Bootstrap 4 (experimen…

### PR DESCRIPTION
**Pull request by GormFrank**
…tal)

### CODE REVIEW

File: site/pages/gcweb-theme/static-header-footer/bootstrap-3.html [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+<title>Static header and footer - Bootstrap 3 - Canada.ca
+</title>
+<meta content="width=device-width,initial-scale=1" name="viewport">
+<meta name="description" content="Static header and footer on Bootstrap 3 with the mega menu. Web Experience Toolkit (WET) includes reusable components for building and maintaining innovative Web sites that are accessible, usable, and interoperable.">
+<!--[if gte IE 9 | !IE ]><!-->
+<link href="https://wet-boew.github.io/themes-dist/GCWeb/assets/favicon.ico" rel="icon" type="image/x-icon">
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+<link rel="stylesheet" href="https://wet-boew.github.io/themes-dist/GCWeb/css/theme.min.css">
+<!--<![endif]-->
+<!--[if lt IE 9]>
```
Comment (duboisp) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-3.html]: We should cleanup old IE conditional markup in this example and replicate that modification.

File: site/pages/gcweb-theme/static-header-footer/bootstrap-3.html [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+					</ul>
+				</nav>
+				<div class="col-xs-6 visible-sm visible-xs tofpg"> <a href="#wb-cont">Top of Page <span class="glyphicon glyphicon-chevron-up"></span></a> </div>
+				<div class="col-xs-6 col-md-3 col-lg-2 text-right"> <img src="https://wet-boew.github.io/themes-dist/GCWeb/assets/wmms-blk.svg" alt="Symbol of the Government of Canada"> </div>
+			</div>
+		</div>
+	</div>
+</footer>
+
+<!--[if gte IE 9 | !IE ]><!-->
```
Comment (duboisp) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-3.html]: Idem

File: site/pages/gcweb-theme/static-header-footer/bootstrap-4.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html><!--[if lt IE 9]><html class="no-js lt-ie9" lang="en" dir="ltr"><![endif]--><!--[if gt IE 8]><!-->
```
Comment (duboisp) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-4.html]: Idem

File: site/pages/gcweb-theme/static-header-footer/bootstrap-4.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+<!--<![endif]-->
+<head>
+<meta charset="utf-8">
+<!-- Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+		wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html -->
+<title>Static header and footer - Bootstrap 4 (Experimental) - No megamenu - Canada.ca
+</title>
+<meta content="width=device-width,initial-scale=1" name="viewport">
+<meta name="description" content="Static header and footer on Bootstrap 4 without the mega menu. This is an experimental feature. Web Experience Toolkit (WET) includes reusable components for building and maintaining innovative Web sites that are accessible, usable, and interoperable.">
+<!--[if gte IE 9 | !IE ]><!-->
```
Comment (duboisp) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-4.html]: Idem

File: site/pages/gcweb-theme/static-header-footer/bootstrap-4.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+						<li class="list-inline-item"><a href="https://www.canada.ca/en/transparency/privacy.html">Privacy</a></li>
+					</ul>
+				</nav>
+				<div class="col-6 offset-6 offset-lg-0 col-lg-3 col-xl-2 text-right"> <img src="https://www.canada.ca/etc/designs/canada/wet-boew/assets/wmms-blk.svg" alt="Symbol of the Government of Canada"> </div>
+			</div>
+		</div>
+	</div>
+</footer>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.js"></script>
```
Comment (duboisp) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-4.html]: Is jQuery needed in this Bootstrap 4 experimental page?

File: site/pages/gcweb-theme/static-header-footer/bootstrap-3.html [hunk 6]
```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html><!--[if lt IE 9]><html class="no-js lt-ie9" lang="en" dir="ltr"><![endif]--><!--[if gt IE 8]><!-->
```
Comment (duboisp) [inline site/pages/gcweb-theme/static-header-footer/bootstrap-3.html]: Remove old IE conditional

File: site/pages/gcweb-theme/static-header-footer/bootstrap-4.html (hunk 5 above)
Comment (GormFrank) [reply site/pages/gcweb-theme/static-header-footer/bootstrap-4.html]: No it is not, I'll remove it!

**Comment by GormFrank** [comment]
Pre-approved during technical review @duboisp.

**Pull request merged**